### PR TITLE
Chore(contributing): introduce contribution guideline

### PR DESCRIPTION
**Pull request by AugustinMauroy**

### CODE REVIEW

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+## 📝 Commit message guidelines
+
+This project follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
+
+Basic rules:
+
+Supported subsystems:
+> build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test
+
+* Commit messages **must** start with a capital letter
```
Comment (Eomm) [inline CONTRIBUTING.MD:25]: Please, relax this section

In the fastify org, we don't apply such strictness

As written in issue #32 
I would just link the main repo's contribution page at https://github.com/fastify/fastify/blob/main/CONTRIBUTING.md
and add more information for new joiners

File: CONTRIBUTING.MD [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+* Commit messages **must not** end with a period `.`
+
+## 📜 Pull Request Policy
+
+### Before merging
```
Comment (Eomm) [inline CONTRIBUTING.MD:31]: At this stage, this is an overhead for me - the only one who is doing it 🥲

File: CONTRIBUTING.MD [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+* There must be no objections after a 24 hour period
+* Fast track is allowed for collaborators.
+
+Each contribution is accepted only if there is no objection to it by a collaborator. During the review, collaborators may request that a specific contributor who is an expert in a particular area give an "LGTM" (looks good to me) before the PR can be merged.
+
+In the case that an objection is raised in a pull request by another collaborator, all collaborators involved should try to arrive at a consensus by addressing the concerns through discussion, compromise, or withdrawal of the proposed change(s).
+
+### When merging
+
+* `squash` pull requests made up of multiple commits
```
Comment (Eomm) [inline CONTRIBUTING.MD:45]: This is already force by the repo setting, so it is impossible to do in a different way

```suggestion
```

File: CONTRIBUTING.MD [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+## Vocabulary
+
+* A **Contributor** is any individual who creates an issue/PR, comments on an issue/PR
+  or contributes in some other way.
+* A **Collaborator** is a contributor with write access to the repository. See [here](#becoming-a-collaborator) on how to become a collaborator.
+
+## Becoming a collaborator
+
+<!-- Write some text about this -->
```
Comment (Eomm) [inline CONTRIBUTING.MD:55]:
```suggestion
We follow the [Fastify organization Governance model](https://github.com/fastify/fastify/blob/main/GOVERNANCE.md#collaborator-nominations) so it is necessary to contribute to this repository in order to become a member.
```

File: CONTRIBUTING.MD [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+[Code of Conduct](https://github.com/fastify/.github/blob/main/CODE_OF_CONDUCT.md)
+
+* [Vocabulary](#vocabulary)
+* [📝 Commit message guidelines](#-commit-message-guidelines)
+* [📜 Pull Request Policy](#-pull-request-policy)
+* [Before merging](#before-merging)
+* [When merging](#when-merging)
+* [Becoming a collaborator](#becoming-a-collaborator)
+* [🐛 Debugging failing checks](#-debugging-failing-checks)
+* [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin-11)
```
Comment (Eomm) [inline CONTRIBUTING.MD:14]: Could we add emojy everywhere or nowhere?
It seems confusing this random thing

**Review by Eomm — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.MD (hunk 1 above)
Comment (AugustinMauroy) [reply CONTRIBUTING.MD:25]: I will embellish this

File: CONTRIBUTING.MD (hunk 2 above)
Comment (AugustinMauroy) [reply CONTRIBUTING.MD:31]: I can help you and you are collaborator so you can do fast track

File: CONTRIBUTING.MD (hunk 5 above)
Comment (AugustinMauroy) [reply CONTRIBUTING.MD:14]: I tried to put some in to make it less boring.

File: CONTRIBUTING.MD (hunk 2 above)
Comment (Eomm) [reply CONTRIBUTING.MD:29]: I think this is over-engineering translated to documentation
 
I would prefer not to commit to this and keep the fastify org rules at https://github.com/fastify/fastify/blob/main/CONTRIBUTING.md#rules

File: CONTRIBUTING.MD [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+## Commit message guidelines
+
+This project follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
+
+Basic rules:
+
+Supported subsystems:
+> build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test
+
+Try to start the commit message with a capital letter. And try to avoid ending it with a period `.`.
```
Comment (Eomm) [inline CONTRIBUTING.MD:24]:
```suggestion
```

File: CONTRIBUTING.MD [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+* Pull Requests must be open for at least 12 hours.
+  * Pull requests might be immediately merged if they contain critical bug fixes, short erratas (e.g. typos from previous PRs) or any critical change that is considered a "showstopper" for the operation of the website.
+    * These kind of PRs should only be done by existing collaborators that have write-access and/or signed-off by administrators/maintainers.
+* There must be no objections after a 24 hour period
+* Fast track is allowed for collaborators.
+
+Each contribution is accepted only if there is no objection to it by a collaborator. During the review, collaborators may request that a specific contributor who is an expert in a particular area give an "LGTM" (looks good to me) before the PR can be merged.
+
+In the case that an objection is raised in a pull request by another collaborator, all collaborators involved should try to arrive at a consensus by addressing the concerns through discussion, compromise, or withdrawal of the proposed change(s).
```
Comment (Eomm) [inline CONTRIBUTING.MD:38]: This repository follows the same rules as a fastify plugin repository. A detailed process is explained at [fastify/CONTRIBUTING.md#rules](https://github.com/fastify/fastify/blob/main/CONTRIBUTING.md#rules)

**Review by Eomm — CHANGES_REQUESTED** [review]
suggested changes to merge the PR

File: CONTRIBUTING.MD (hunk 7 above)
Comment (Eomm) [inline CONTRIBUTING.MD:38]:
```suggestion
This repository follows the same rules as a fastify plugin repository. A detailed process is explained at [fastify/CONTRIBUTING.md#rules](https://github.com/fastify/fastify/blob/main/CONTRIBUTING.md#rules)
```

**Review by Eomm — APPROVED** [review]

**Pull request merged**